src/NuGet/Microsoft.Net.Compilers.netcore.nuspec

```diff
@@ -60,6 +60,8 @@
         </dependencies>
     </metadata>
     <files>
+        <file src="$emptyDirPath$/_._" target="ref/dotnet" />
```

Comment:
Was it adding the APIs that are directly in csc.exe and vbc.exe? If so then these placeholders will help there but if it is prompting the dependencies of this package into the compile graph which I suspect is the case here then these placeholders won't have any affect on that. If you wanted to block the promotion then you need the exclude=compile attribute on the dependency.